tests/unit/conftest.py

```diff
@@ -64,24 +64,32 @@ def patch_all(tmp_path: Path):
 
 
 @pytest.fixture(autouse=True)
-def mock_worker_lightkube_client(request):
-    """Global mock for the Worker's lightkube client to avoid lightkube calls."""
+def mock_lightkube_client(request):
+    """Global mock for lightkube client to avoid lightkube calls in both Worker and Coordinator."""
     # Skip this fixture if the test has explicitly disabled it.
-    # To use this feature in a test, mark it with @pytest.mark.disable_worker_lightkube_client_autouse
-    if "disable_worker_lightkube_client_autouse" in request.keywords:
+    # To use this feature in a test, mark it with @pytest.mark.disable_lightkube_client_autouse
+    if "disable_lightkube_client_autouse" in request.keywords:
         yield
     else:
-        with patch("coordinated_workers.worker.Client") as mocked:
-            yield mocked
+        with ExitStack() as stack:
```

Comment:
Not suggestion: i'd keep this fixture just for the worker, and have a second fixture for coordinator. yes, there's a little duplication, but it's trivial and then the return here is explicitly instead of a dict with untyped keys.

Reply:
same comment for the other patch below, too